dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -197,7 +197,17 @@ public Connection getConnection() throws SQLException {
         return hs.toArray(array);
     }
 
-    protected abstract Pattern getParameterPattern();
+    protected String defaultParamPatternString = "";
+
+    protected String getParamPatternString() {
+        return Options.get(Options.OPTION_PARAMETER_PATTERN).equals("") ?
+            defaultParamPatternString : Options.get(Options.OPTION_PARAMETER_PATTERN);
+    }
+
+    //protected abstract Pattern getParameterPattern();
```

Comment:
Isn't that comment redundant? Maybe we rather refer to `defaultParamPatternString` attribute as a comment here?